packages/flutter/lib/src/services/raw_keyboard_macos.dart

```diff
@@ -156,6 +140,9 @@ class RawKetEventDataMacOs extends RawKeyEventData {
   KeyboardSide getModifierSide(ModifierKey key) => KeyboardSide.any;
 
   // Modifier key masks.
+  // See also:
+  //
+  //   * [Apple's NSEvent documentation](https://developer.apple.com/documentation/appkit/nseventmodifierflags?language=objc)
```

Comment:
Since this isn't a dartdoc comment (///), you don't need to add the markdown link here.

If you want to keep the URL, that's fine, but it shouldn't be markdown formatted.